### PR TITLE
Added knative-extensions

### PR DESCRIPTION
**Pull request by davidhadas**
Add support for `knative-extensions`
removed `knative-sandbox`

### CODE REVIEW

**Comment by davidhadas** [comment]
Any reason why test does not pass?

File: library.sh [hunk 1]
```diff
@@ -46,6 +46,7 @@ readonly REPO_ROOT_DIR
 function __resolveRepoName() {
   local repoName
   repoName="$(basename "${1:-$(git rev-parse --show-toplevel)}")"
+  repoName="${repoName#knative-extension-}" # Remove knative-extension- prefix if any
```
Comment (davidhadas) [inline library.sh:49]:
```suggestion
  repoName="${repoName#knative-extensions-}" # Remove knative-extensions- prefix if any
```

**Review by upodroid — CHANGES_REQUESTED** [review]
It can replace knative-sandbox instead

File: library.sh [hunk 2]
```diff
@@ -46,6 +46,7 @@ readonly REPO_ROOT_DIR
 function __resolveRepoName() {
   local repoName
   repoName="$(basename "${1:-$(git rev-parse --show-toplevel)}")"
+  repoName="${repoName#knative-extensions-}" # Remove knative-extensions- prefix if any
   repoName="${repoName#knative-sandbox-}" # Remove knative-sandbox- prefix if any
```
Comment (davidhadas) [inline library.sh:50]:
```suggestion
```

**Review by upodroid — APPROVED** [review]
LGTM

**Pull request merged**